code_schemes/Change.json

```diff
@@ -0,0 +1,78 @@
+{
+  "Codes": [
+    {
+      "CodeID": "code-baec502746d5",
+      "CodeType": "Normal",
+      "DisplayText": "Test",
```

Comment:
If you're going to include a test code, then take care to think through the implications first. If you delete this code from the scheme but still have data with this label, then I suspect Coda will give you scary warnings and the pipeline will crash with KeyErrors.